components/ui/eventsList/index.tsx

```
@@ -202,7 +250,7 @@ function EventsList({ events: serverEvents = [] }) {
   return (
     <>
       <Script
-        id={`${place || "catalunya"}-${byDate || "all"}-script`}
+        id={`${label || "catalunya"}-${byDate || "all"}-script`}
         type="application/ld+json"
         dangerouslySetInnerHTML={{ __html: JSON.stringify(jsonEvents) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)